sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -874,6 +875,96 @@ class AdaptiveQueryExecSuite
     }
   }
 
+  test("SPARK-44426: Optimize adaptive skew join for ExistenceJoin") {
+    withSQLConf(
+      SQLConf.AUTO_BROADCASTJOIN_THRESHOLD.key -> "-1",
+      SQLConf.COALESCE_PARTITIONS_MIN_PARTITION_NUM.key -> "1",
+      SQLConf.SHUFFLE_PARTITIONS.key -> "100",
+      SQLConf.SKEW_JOIN_SKEWED_PARTITION_THRESHOLD.key -> "800",
+      SQLConf.ADVISORY_PARTITION_SIZE_IN_BYTES.key -> "800") {
+      withTempView("skewData1", "skewData2", "skewData3", "skewData4") {
```

Comment:
Can we simplify and clear the test a bit? We can make tests using one join that one side is a skewed table and other side is not. Then we can switch it's side to verify what happens if the skewed side is on left or right.

Reply:
> Can we simplify and clear the test a bit? We can make tests using one join that one side is a skewed table and other side is not. Then we can switch it's side to verify what happens if the skewed side is on left or right.

@ulysses-you Have been modified. Thanks for reviewing